Source/Plugins/Core/com.equella.core/js/tsrc/search/SearchPage.tsx

```diff
@@ -436,8 +436,9 @@ const SearchPage = ({ updateTemplate }: TemplateUpdateProps) => {
       false
     );
 
-  const handleRawModeChanged = (rawMode: boolean) =>
-    search({ ...searchPageOptions, rawMode: rawMode });
+  const handleWildcardModeChanged = (wildcardMode: boolean) =>
+    // We use `rawMode` which is the logic NOT of `wildcardMode` at lower level.
```

Comment:
Sorry, I'm not sure what that is saying, but I propose:

```suggestion
    // `wildcardMode` is a presentation concept, in the lower levels its inverse is the value for `rawMode`
```